nuxeo-drive-client/tests/test_engine_dao.py

```diff
@@ -15,8 +15,7 @@
 class EngineDAOTest(unittest.TestCase):
 
     def _get_default_db(self, name='test_engine.db'):
-        nxdrive_path = os.path.dirname(nxdrive.__file__)
-        return os.path.join(nxdrive_path, 'tests', 'resources', name)
+        return os.path.join(os.path.dirname(__file__), 'resources', name)
```

Comment:
`self.location` is only defined in `IntegrationTestCase` and `UnitTestCase`. This particular test file does not use one of them.